js_modules/dagit/packages/core/src/assets/AutoMaterializePolicyPage/AutomaterializeLeftPanel.tsx

```diff
@@ -65,38 +65,49 @@ export const AutomaterializeLeftList = (props: ListProps) => {
         const isSelected = selectedEvaluation?.evaluationId === evaluation.evaluationId;
         if (evaluation.__typename === 'no_conditions_met') {
           return (
-            <EvaluationListItem
-              key={`skip-${evaluation.evaluationId}`}
-              onClick={() => {
-                onSelectEvaluation(evaluation);
-              }}
-              $selected={isSelected}
-            >
-              <Box flex={{direction: 'column', gap: 4}} style={{width: '100%'}}>
-                <div>
-                  {evaluation.startTimestamp ? (
-                    evaluation.amount === 1 ? (
-                      '1 evaluation'
-                    ) : (
-                      `${compactNumber(evaluation.amount)} evaluations`
-                    )
-                  ) : (
-                    <>
-                      {evaluation.endTimestamp === 'now' ? (
-                        'Before now'
+            <>
+              <EvaluationListItem
+                key={`skip-${evaluation.evaluationId}`}
+                onClick={() => {
+                  onSelectEvaluation(evaluation);
+                }}
+                $selected={isSelected}
+              >
+                <Box flex={{direction: 'column', gap: 4}} style={{width: '100%'}}>
+                  <div>
+                    {evaluation.startTimestamp ? (
+                      evaluation.amount === 1 ? (
+                        '1 evaluation'
                       ) : (
-                        <>
-                          Before <TimestampDisplay timestamp={evaluation.endTimestamp} />
-                        </>
-                      )}
-                    </>
-                  )}
-                </div>
-                <Caption color={isSelected ? Colors.Blue700 : Colors.Gray700}>
-                  No conditions met
-                </Caption>
-              </Box>
-            </EvaluationListItem>
+                        `${compactNumber(evaluation.amount)} evaluations`
+                      )
+                    ) : (
+                      <>
+                        {evaluation.endTimestamp === 'now' ? (
+                          'Before now'
+                        ) : (
+                          <>
+                            Before <TimestampDisplay timestamp={evaluation.endTimestamp} />
+                          </>
+                        )}
+                      </>
+                    )}
+                  </div>
+                  <Caption color={isSelected ? Colors.Blue700 : Colors.Gray700}>
+                    No conditions met
+                  </Caption>
+                </Box>
+              </EvaluationListItem>
+              {!evaluation.startTimestamp ? (
```

Comment:
I think you could put this `Box` at the end of the list, after the `map()` call, instead of rendering it as part of a list item child. You could just look at the last item in `evaluationsIncludingEmpty`, and if there's no `startTimestamp` on it, render this `Box`.

Reply:
I actually decided to just show it on every page- it's relevant regardless of if you've scrolled to the last page